types/applications/application.d.ts

```diff
@@ -18,10 +18,8 @@ declare let _maxZ: number;
  *   renderApplication
  *   closeApplication
  *   getApplicationHeaderButtons
- *
- * @typeParam T - the type of the data used to render the inner template
  */
-declare class Application<T = object> {
+declare class Application {
```

Comment:
Should this be `abstract`?

Reply:
I know I suggested it but I'm not 100% sure. It's not marked `@abstract` in `foundry.js` but clearly isn't meant to be instantiated directly... @am-a @kmoschcau any opinions on that?

Reply:
If it doesn't have an abstract method, I'd say no.

Reply:
OK, I'll leave it as it is for now.